task3/src/models/user.model.ts

```diff
@@ -0,0 +1,44 @@
+import { Model, DataTypes } from 'sequelize';
+import { v4 as uuidv4 } from 'uuid';
+
+import Database from '../data-access/database';
+
+export default class UserModel extends Model {
+    public id!: string;
+    public login!: string;
+    public password!: string;
+    public age!: number;
+    public isDeleted!: boolean;
+}
+
+UserModel.init(
+    {
+        id: {
+            type: DataTypes.UUID,
+            primaryKey: true,
+            allowNull: false,
+            defaultValue: () => uuidv4(),
+        },
+        login: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
+        password: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
+        age: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+        },
+        isDeleted: {
```

Comment:
You may update the code to make a real deletion of data from db:)

Reply:
Deleted